### ISSUE TITLE
Java client lib: clarify intent of ExperimentClient class

### ISSUE THREAD

**Issue by jerith666** [CarnegieLearningWeb/educational-experiment-service-client-libs#28]
This class lives in the `example` package, which implies to me that it's just an illustration of how a user of this library should use the public API of this library, which is `ExperimentServiceAPI`.

However, in `java/README.md`, the implication is that a user of this library would interact with `ExperimentClient`, not `ExperimentServiceAPI`.

Please clarify which is intended.

**Comment by dhavalocked** [comment]
`ExperimentClient` is actual class where all the methods are implemented. User will make a constructor of this class and call different methods. (  **This class will be used by other services** who wants to use this library)

`ExperimentServiceAPI` is just an interface where we set Request type, retry policy, URL,  define expected params and expected response. This class is used by  `ExperimentClient` class to build URLs and pass params. 

Example:  Method of `ExperimentServiceAPI`

```
  @Retry
  @POST("api/groupmembership")
  Call<InitRequest> setGroupMemebership(@Body InitRequest initRequest);
```

Now `ExperimentClient ` will make use of this class and build a service using service class.

```
ExperimentServiceAPI client = ServiceGenerator.createService(ExperimentServiceAPI.class);
client.setGroupMemebership(initRequest).enqueue( ... )
```

**Comment by jerith666** [comment]
Okay, thanks.  Then I think the resolution here would be to move the `ExperimentClient` to a more-official-sounding package, perhaps `client`?

**Comment by dhavalocked** [comment]
Fixed in #34

**Comment by jerith666** [comment]
yep, #34 fixes the package name -- thanks!
